src/en/discover/case-studies/hyperdrive-ceph-in-a-box/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "HyperDrive: Ceph-in-a-box"
+image: "/assets/bitmaps/ceph-in-a-box.jpg"
```

Comment:
Please see my other comment about a naming convention with a prefix.

You'll need to update this if you add the prefix.

Reply:
Have now re-named the image file to use the prefix as recommended.

Reply:
@fishoutoforder Thanks. Now the asset filename's been changed, we'll need to update the `image` path here too:

```suggestion
image: "/assets/bitmaps/photo-ceph-in-a-box.jpg"
```